liteflow-labs/starter-kit
# SSR improvements

Dependency: https://github.com/liteflow-labs/starter-kit/pull/241

### Description

Update SSR and use `getDataTree` to resolve all graphql queries
Improve the `useAccount` to have better compatibility on the server side
Fix cache issue with collection composed key

No more `getServerSideProps` 🎉 

### How to test

You can just run the different website pages with and without a refresh and check that the queries to get the data in the network panel do not have the pages' queries.
Also, you can use the settings `disable javascript` to really see the ssr.

### Checklist
